src/functions.jl

```diff
@@ -254,18 +254,24 @@ function sum_dict(kvs::Vector{Pair{K, V}}) where {K, V}
     d
 end
 
-_dicts(f::ScalarAffineFunction) = sum_dict(Pair.(f.variables, f.coefficients))
-_dicts(f::VectorAffineFunction) = sum_dict(Pair.(tuple.(f.outputindex, f.variables), f.coefficients))
-
+function _pair(t::Union{VectorAffineTerm, VectorQuadraticTerm})
```

Comment:
Could you add some comments explaining what these methods do? Underscore isn't a license to provide no comments :)